backend/server.js

```diff
@@ -1,30 +1,147 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
+import crypto from "crypto";
+import bcrypt, { genSaltSync } from "bcrypt";
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/authAPI"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
+// Theare are env varaibles that are passed in when the backend server runs. This way we don't store the database username or password
+const userDB = process.env.userDB;
+const passDB = process.env.passDB;
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
+// Local host url
+// mongodb://localhost/authAPI
+
+const mongoUrl =
+  process.env.MONGO_URL ||
```

Comment:
We are curious why you choose to place the connection string here, on the right side of the OR sign?